### PR TITLE
[1LP][RFR] Adding playbooks for automate test cases

### PR DESCRIPTION
**Pull request by ganeshhubale**
- This PR adds the playbooks which are required for testing automate test case.
- This is helping to test RFE(1678132, 1678135)

### CODE REVIEW

**Comment by digitronik** [comment]
@ganeshhubale please avoid name like `playbook1` give specific names. even check file ending

File: get_facts_from_juniper.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- name: Collect facts from Juniper device
```
Comment (kedark3) [inline get_facts_from_juniper.yml]: @ganeshhubale this looks good, although do we have any juniper devices to run this against?

**Review by kedark3 — COMMENTED** [review]
Overall playbooks look good, one question though, @ganeshhubale do you need to execute these playbooks in automation? 
Or just add the playbooks and check if they exists?

File: get_facts_from_juniper.yml (hunk 1 above)
Comment (sbulage) [reply get_facts_from_juniper.yml]: @kedark3 I have asked @ganeshhubale to upload this Playbook. I have plan to automate Ansible Network credentials.

**Comment by sbulage** [comment]
> Overall playbooks look good, one question though, @ganeshhubale do you need to execute these playbooks in automation?
> Or just add the playbooks and check if they exists?

The playbooks starts with `get*` will require to automate few Ansible Tests.

**Comment by sbulage** [comment]
Rather creating multiple PRs against this Repository Thought to ask @ganeshhubale add few Playbooks there. which reduces extra efforts. :smile:

**Review by sbulage — APPROVED** [review]
LGTM :+1:

**Comment by ganeshhubale** [comment]
> Overall playbooks look good, one question though, @ganeshhubale do you need to execute these playbooks in automation?
> Or just add the playbooks and check if they exists?

- Yes, these playbooks are required while automating test case(```test_variable_pass_method_playbook``` and ```test_variable_pass_successive_playbook```).
- I have added these playbooks as per your suggestion. So that I need not to depend on [```billy's``` repo.](https://github.com/ManageIQ/integration_tests/pull/9104#discussion_r307477191)

File: get_facts_from_openstack_workaround.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+- name: Collect facts from Openstack instances - workaround
+  hosts: all
+  vars:
+  tasks:
+  - include_vars:
+      # file: "{{ lookup('file', '/var/tmp/openstack_creds') }}"
```
Comment (digitronik) [inline get_facts_from_openstack_workaround.yml]: I think you can remove this ^^

File: get_facts_from_openstack_workaround.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      file: '/var/tmp/openstack_creds'
+      name: os_creds
+  - os_server_facts:
+      timeout: 60
+    environment:
+      OS_USERNAME: "{{ os_creds.clouds.osp11.auth.username }}"
+      OS_PASSWORD: "{{ os_creds.clouds.osp11.auth.password }}"
+      OS_AUTH_URL: "{{ os_creds.clouds.osp11.auth.auth_url }}"
+      OS_PROJECT_NAME:          "{{ os_creds.clouds.osp11.auth.project_name }}"
```
Comment (digitronik) [inline get_facts_from_openstack_workaround.yml]: I would like to fallow code spacing in playbooks as well. I know this for testing but good have.

File: get_facts_from_openstack_workaround.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+- name: Collect facts from Openstack instances - workaround
+  hosts: all
+  vars:
+  tasks:
+  - include_vars:
+      # file: "{{ lookup('file', '/var/tmp/openstack_creds') }}"
+      file: '/var/tmp/openstack_creds'
```
Comment (kedark3) [inline get_facts_from_openstack_workaround.yml]: Do you create this file when you execute the playbook?

Comment (ganeshhubale) [reply get_facts_from_openstack_workaround.yml]: @sbulage 
Please add your view on this question.

Comment (kedark3) [reply get_facts_from_openstack_workaround.yml]: @sbulage @ganeshhubale I really want to understand if each of the playbook you are adding in this PR is going to be executed? 
If so, have you tested it ? Can you please share the results? Thanks.

Comment (ganeshhubale) [reply get_facts_from_openstack_workaround.yml]: hello @kedark3 
I have automated test case which uses these playbooks.
Please have a look: https://github.com/ManageIQ/integration_tests/pull/9222

File: input_vars_third.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+  tasks:
+  - name: "playbook3a - testing input vars"
+    set_stats:
+      data:
+        pb_var3a: "{{ input3a }}"
+        pb_var3b: "{{ input3b }}"
+
+  - debug: msg="Hello, {{ input3a }} and {{ input3b }}."
+  - debug: msg="Hello, State_vars from playbook2 are {{ pb_var2a }} and {{ pb_var2b }}."
```
Comment (kedark3) [inline input_vars_third.yml]: @ganeshhubale did you want to  say `State_vars` or `Stat_vars` ? Asking since you are setting `stats`

**Review by kedark3 — COMMENTED** [review]
One comment only. Looks good otherwise. Please point me to PRT where these playbooks are used in your automation if possible.

**Review by digitronik — APPROVED** [review]
LGTM :+1:

**Review by izapolsk — APPROVED** [review]

**Pull request merged**